frontend/mobilityai/.idea/assetWizardSettings.xml

```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This file shouldn't be committed, see https://github.com/pgunasekara/4zp6/pull/77#issuecomment-457843196

Reply:
Should be fixed with the new gitignore